scripts/actions/check-for-outdated-translations.js

```diff
@@ -0,0 +1,114 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('node-fetch');
+const frontmatter = require('@github-docs/frontmatter');
+const parseLinkHeader = require('parse-link-header');
+
+const checkArgs = require('./utils/check-args');
+const { prop } = require('../utils/functional');
+const { ADDITIONAL_LOCALES } = require('../utils/constants');
+
+const doI18nFilesExist = (fileName, locales) => {
+  const i18nPrefix = path.join(process.cwd(), 'src/i18n/content');
+  const baseFileName = fileName.replace('src/content/', '');
+
+  return locales
+    .map((locale) => {
+      const filePath = path.join(i18nPrefix, locale, baseFileName);
+      const fileExists = fs.existsSync(filePath);
+      return fileExists ? filePath : null;
+    })
+    .filter(Boolean);
+};
+
+const fetchFilesFromGH = async (url) => {
+  let files = [];
+  let nextPageLink = url;
+
+  while (nextPageLink) {
+    const resp = await fetch(nextPageLink, {
+      headers: { authorization: `token ${process.env.GITHUB_TOKEN}` },
+    });
+    const page = await resp.json();
```

Comment:
We might want to add some error handling around this. If a non-200 status code is returned, the response object will not have the `.json()` method. Erroring out and letting the catch handle it works, but we lose some visibility into the actual error that the API is returning.

Adding something like this before line 32 should do the trick.
```javascript
if (!resp.ok) {
  throw new Error(`Github API returned status ${resp.code} - ${resp.message}`);
}
```